backend/main.py

```diff
@@ -88,59 +100,44 @@ async def retrieve_available_lights():
 @fast_app.post("/selected-lights")
 def store_selected_lights(lights: List[str]):
     if not lights:
-        raise HTTPException(status_code=400, detail="Selected lights list is required")
+        raise HTTPException(status_code=400, detail='\"lights\" list is required.')
 
     try:
-        redis_client.set("lights", ",".join(lights))
+        redis_client.set(constants.REDIS_SELECTED_LIGHTS_KEY, ",".join(lights))
 
-        response = StandardResponse(
+        return StandardResponse(
             success=True, message="Selected lights list stored in Redis"
         )
 
     except redis.exceptions.RedisError as redis_err:
         raise HTTPException(status_code=500, detail=f"Redis Error: {str(redis_err)}")
 
-    return response
-
 
 @fast_app.put("/start-spotihue")
 async def start_spotihue(lights: List[str] = None):
-    try:
-        if not lights:
-            # stored_lights = redis_client.get("lights")
-            # if stored_lights:
-            #     lights = stored_lights.split(",")
-            # else:
-            raise ValueError("Empty list of lights")
+    if not lights:
+        raise HTTPException(status_code=400, detail='\"lights\" list is required.')
 
-        spotihue_status = redis_client.get("spotihue")
-        print(spotihue_status)
```

Comment:
This was supposed to handle if it was already running or not and you press the run button again, but it didn't really work... Should we leave it commented out instead of deleting?

Reply:
I can leave it if you prefer! For myself, I don't need it - I left a "TODO: make this idempotent" comment below so I remember to re-implement this ^ logic and troubleshoot it more...

Reply:
Perfect. Let's delete it.